cmd/dfget/app/root.go

```diff
@@ -94,34 +108,58 @@ func initProperties() {
 	if cfg.ClientQueueSize == 0 {
 		cfg.ClientQueueSize = properties.ClientQueueSize
 	}
+}
 
+// transParams trans the user-friendly parameter formats
+// to the format corresponding to the `Config` struct.
+func transParams() error {
 	cfg.Filter = transFilter(filter)
 
-	var err error
-	cfg.LocalLimit, err = transLimit(localLimit)
-	util.PanicIfError(err, "convert locallimit error")
-	cfg.TotalLimit, err = transLimit(totalLimit)
-	util.PanicIfError(err, "convert totallimit error")
+	lLimit, err := transLimit(localLimit)
```

Comment:
use named return value, these codes could be written:

```go
if cfg.LocalLimit, err = transLimit(localLimit); err != nil {
    ...
}
```

Reply:
Done.